server/get_payload.go

```diff
@@ -111,18 +136,18 @@ func (m *BoostService) getPayload(log *logrus.Entry, signedBlindedBeaconBlockByt
 	originalBid := m.bids[bidKey(slot, blockHash)]
 	m.bidsLock.Unlock()
 	if originalBid.response.IsEmpty() {
-		log.Error("no bid for this getPayload payload found, was getHeader called before?")
+		log.Error("no bid for this payload found, was getHeader called before?")
```

Comment:
```suggestion
		log.Warn("no bid for this payload found, was getHeader called before?")
```

a warn is better here, and it's common for this to happen in DV setups